modules/oe_webtools_etrans/src/Plugin/Block/UnifiedEtransBlock.php

```diff
@@ -79,7 +79,7 @@ public static function create(ContainerInterface $container, array $configuratio
   /**
    * {@inheritdoc}
    */
-  public function access(AccountInterface $account, $return_as_object = FALSE) {
+  public function blockAccess(AccountInterface $account, $return_as_object = FALSE) {
```

Comment:
The blockAccess() has no second parameter. See: BlockPluginTrait::blockAccess()

Reply:
Of course, if I'm checking access(). Updated.